src/Simplex/Chat/Store/Files.hs

```diff
@@ -536,6 +538,23 @@ createRcvGroupFileTransfer db userId GroupMember {groupId, groupMemberId, localD
       (fileId, FSNew, fileConnReq, fileInline, rcvFileInline, groupMemberId, rfdId, currentTs, currentTs)
   pure RcvFileTransfer {fileId, xftpRcvFile, fileInvitation = f, fileStatus = RFSNew, rcvFileInline, senderDisplayName = c, chunkSize, cancelled = False, grpMemberId = Just groupMemberId, cryptoArgs = Nothing}
 
+createRcvDirectFileTransfer :: DB.Connection -> UserId -> CryptoFile -> Int64 -> Word32 -> ExceptT StoreError IO Int64
```

Comment:
```suggestion
createRcvFileTransferNoChat :: DB.Connection -> UserId -> CryptoFile -> Int64 -> Word32 -> ExceptT StoreError IO Int64
```
or `createRcvNoChatFileTransfer`, or `createRcvStandaloneFileTransfer`.

"Direct" is user for direct (contact) conversations all over the code, this would be very confusing. The function two functions above "createRcvFileTransfer" would be called "direct", it doesn't have it in name probably because it's so old that at the time files weren't supported in groups (or maybe groups didn't exist at all).